src/conv.rs

```diff
@@ -234,7 +244,40 @@ pub fn map_device_descriptor<'a>(
     }
 }
 
-pub fn map_limits(limits: native::WGPULimits) -> wgt::Limits {
+pub fn map_pipeline_layout_descriptor<'a>(
+    des: &native::WGPUPipelineLayoutDescriptor,
+    extras: Option<&native::WGPUPipelineLayoutExtras>,
+) -> wgc::binding_model::PipelineLayoutDescriptor<'a> {
+    let mut push_constant_ranges: Vec<PushConstantRange> = Vec::new();
+
+    if let Some(extras) = extras {
+        for i in 0..extras.pushConstantRangeCount {
```

Comment:
we should just construct the slice here, i.e. `let raw_push_constant_ranges = slice::from_raw_parts(...);`